vllm/model_executor/models/utils.py

```diff
@@ -238,6 +238,90 @@ def load_weights(
         return autoloaded_weights
 
 
+class OffloadedTensorMode(TorchDispatchMode):
+
+    def __torch_dispatch__(self, func, types, args=(), kwargs=None):
+        kwargs = {} if kwargs is None else kwargs
+        tensor = func(*args, **kwargs)
+
+        if (func is torch.ops.aten.empty.memory_format
+                and tensor.device != "cpu"):
```

Comment:
maybe use `torch.device("cpu")` instead of `"cpu"`?

Reply:
hmm `tensor.device != "cpu` should generally be ok